mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -394,137 +395,179 @@ PopulateParamsAccel::obtainTuningParameters(RockGemmWrapperInterface op,
-  {64, 128, 2, 32, 64, 8, 1, true, true},
-  {64, 128, 2, 32, 64, 4, 1, true, true},
-  {64, 64, 8, 32, 32, 4, 1, true, true},
-  {64, 64, 4, 32, 32, 8, 1, true, true},
-  {64, 64, 4, 32, 32, 4, 1, true, true},
-  {64, 64, 4, 16, 64, 8, 1, true, true},
-  {64, 64, 4, 16, 64, 4, 1, true, true},
-  {64, 64, 2, 32, 32, 8, 1, true, true},
-  {32, 128, 4, 32, 32, 4, 1, true, true},
-  {32, 64, 4, 32, 64, 4, 1, true, true},
-  {32, 64, 4, 8, 64, 4, 1, true, true},
-  {32, 64, 8, 32, 32, 4, 1, true, true},
-  {32, 64, 8, 8, 64, 4, 1, true, true},
-  {32, 64, 2, 32, 32, 8, 1, true, true},
-  {32, 64, 2, 8, 64, 4, 1, true, true},
-  {32, 32, 8, 16, 16, 8, 1, true, true},
-  {32, 32, 8, 16, 16, 4, 1, true, true},
-  {32, 32, 4, 16, 16, 8, 1, true, true},
-  {16, 64, 8, 16, 16, 8, 1, true, true},
-  {16, 64, 8, 16, 16, 4, 1, true, true},
-  {16, 32, 8, 16, 16, 8, 1, true, true},
-  {16, 32, 8, 16, 16, 4, 1, true, true},
-  {16, 16, 8, 16, 16, 8, 1, true, true},
-  {16, 16, 4, 16, 16, 4, 1, true, true},
-  {8, 64, 8, 4, 64, 8, 1, true, true},
-  {4, 64, 16, 4, 64, 1, 1, true, true},
-  {4, 64, 2, 4, 64, 8, 1, true, true}
+  // M/block N/block K/block M/wave mnkPerXdl kPack splitKFactor forceUnroll bCopyMore
+  {128, 256, 4, 64, 3202, 4, 1, true, true},
+  {128, 128, 8, 64, 3202, 1, 1, true, true},
+  {128, 128, 8, 64, 3202, 1, 1, true, true},
+  {128, 128, 8, 64, 3202, 4, 1, true, true},
+  {128, 128, 4, 64, 3202, 4, 1, true, true},
+  {128, 128, 4, 64, 3202, 8, 1, true, true},
+  {128, 128, 4, 64, 3202, 4, 1, true, true},
+  {128, 128, 2, 64, 3202, 8, 1, true, true},
+  {128, 128, 2, 64, 3202, 8, 1, true, true},
+  {128, 128, 2, 64, 3202, 4, 1, true, true},
+  {64, 256, 8, 64, 3202, 1, 1, true, true},
+  {64, 256, 4, 64, 3202, 4, 1, true, true},
+  {64, 128, 4, 32, 3202, 4, 1, true, true},
+  {64, 128, 2, 32, 3202, 8, 1, true, true},
+  {64, 128, 2, 32, 3202, 4, 1, true, true},
+  {64, 64, 8, 32, 3202, 4, 1, true, true},
+  {64, 64, 4, 32, 3202, 8, 1, true, true},
+  {64, 64, 4, 32, 3202, 4, 1, true, true},
+  {64, 64, 4, 16, 1604, 8, 1, true, true},
+  {64, 64, 4, 16, 1604, 4, 1, true, true},
+  {64, 64, 2, 32, 3202, 8, 1, true, true},
+  {32, 128, 4, 32, 3202, 4, 1, true, true},
+  {32, 64, 4, 32, 3202, 4, 1, true, true},
+  {32, 64, 4, 8, 401, 4, 1, true, true},
+  {32, 64, 8, 32, 3202, 4, 1, true, true},
+  {32, 64, 8, 8, 401, 4, 1, true, true},
+  {32, 64, 2, 32, 3202, 8, 1, true, true},
+  {32, 64, 2, 8, 401, 4, 1, true, true},
+  {32, 32, 8, 16, 1604, 8, 1, true, true},
+  {32, 32, 8, 16, 1604, 4, 1, true, true},
+  {32, 32, 4, 16, 1604, 8, 1, true, true},
+  {16, 64, 8, 16, 1604, 8, 1, true, true},
+  {16, 64, 8, 16, 1604, 4, 1, true, true},
+  {16, 32, 8, 16, 1604, 8, 1, true, true},
+  {16, 32, 8, 16, 1604, 4, 1, true, true},
+  {16, 16, 8, 16, 1604, 8, 1, true, true},
+  {16, 16, 4, 16, 1604, 4, 1, true, true},
+  {8, 64, 8, 4, 401, 8, 1, true, true},
+  {4, 64, 16, 4, 401, 1, 1, true, true},
+  {4, 64, 2, 4, 401, 8, 1, true, true}
```

Comment:
... could we just go ahead and declare `v3` instead of pulling this parameter packing trick?

Reply:
We can... 

Just want to make sure -- you are asking for MFMA specific perfconfig format ?
i.e. v3mfma: xx, xx, ... , xx .